i18n/en.json

```diff
@@ -3505,11 +3505,11 @@
   },
   {
     "id": "model.config.is_valid.webrtc_gateway_admin_url.app_error",
-    "translation": "WebRTC Gateway Admin Url must be set a valid URL and start with http:// or https://."
+    "translation": "WebRTC Gateway Admin URL must be set a valid URL and start with http:// or https://."
   },
   {
     "id": "model.config.is_valid.webrtc_gateway_ws_url.app_error",
-    "translation": "WebRTC Gateway Websocket Url must be a valid URL and start with ws:// or wss://."
+    "translation": "WebRTC Gateway Websocket URL must be a valid URL and start with ws:// or wss://."
```

Comment:
Propose: "WebRTC Gateway Websocket URL must be a valid URL starting with ws:// or wss://"